src/main/java/com/northconcepts/datapipeline/mailchimp/ReadMailchimpContacts.java

```diff
@@ -0,0 +1,25 @@
+package com.northconcepts.datapipeline.mailchimp;
+
+import com.northconcepts.datapipeline.mailchimp.list.ListMember;
+import com.northconcepts.datapipeline.mailchimp.list.ListMemberStatus;
+import com.northconcepts.datapipeline.mailchimp.list.ListMembers;
+
+public class ReadMailchimpContacts {
+    private static final String apiKey = "api-key";
+    private static final String listId = "list-id";
+
+    private static final MailChimpClient client = MailChimpClient.Proxy.get(apiKey);
```

Comment:
Can we use `MailChimpListMemberReader` in a DP job instead of the Client API?

You're welcome to call this example `ReadMailchimpContactsUsingClientAPI`.

Reply:
Done